sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_BuildModel.md

```diff
@@ -0,0 +1,59 @@
+# Build a model
+
+This sample demonstrates how to build a model with your own data. A custom model can output structured data that includes the relationships in the original document. After you build the model, you can test and retrain it and eventually use it to reliably extract data from more documents according to your needs.
```

Comment:
I am not sure if we can "retrain" an already built model, can we?

Reply:
we can't, u are rgith. this is misleading. I guess the idea was to say keep creating models until u are happy.
I will delete this